ignite/handlers/running_average.py

```diff
@@ -0,0 +1,66 @@
+from ignite.metrics import Metric
+from ignite.engine import Events
+
+
+class RunningAverage(object):
```

Comment:
Can we not make this inherit from `Metric`? It feels as though the running average is also a metric (of metrics)?

Reply:
yes you are right, as discussed in the issue at first it was implemented as a class inherited from Metric. But then I did not find any advantage of doing that...
See #264

Reply:
Agreed. Conceptually, I think it makes sense for this to be with other "metrics."